### DISCUSSION TITLE
Solodit - Unable to open new tab on certain issue (dead link)

### DISCUSSION THREAD

**Discussion by xxlongzaixx** [solodit/feedback#25] [Bug Report] [answered]
<img width="1278" alt="solodit-dead-link" src="https://github.com/solodit/feedback/assets/106563476/7e4ac89c-a589-4b22-a2aa-7e92c8137353">

**Comment by wang1252** [comment, marked as answer]
@xxlongzaixx 

Thank you for your report. 
But the link is valid and we can navigate correctly.
Could you try again?

**Reply by xxlongzaixx** [reply]
cool, it is working fine now, thanks
